£Lerror_reason¢DtypeEvalueY=HTTPSConnectionPool(host='www.ics.uci.eduhttps', port=443): Max retries exceeded with url: //www.ics.uci.edu/community/news/view_news?id=1440 (Caused by ConnectTimeoutError(<urllib3.connection.VerifiedHTTPSConnection object at 0x0000000004AB4198>, 'Connection to www.ics.uci.eduhttps timed out. (connect timeout=2)'))Ihttp_code¢DtypeEvalueÛQdownload_complete¢DtypeEvalueı